[2024-04-14 18:45:26,722] INFO _log() L96    * Restarting with stat - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:45:31,050] WARNING _log() L96    * Debugger is active! - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:45:31,056] INFO _log() L96    * Debugger PIN: 730-829-146 - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:45:48,318] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:45:48] "POST /obs HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:46:20,687] INFO _log() L96    * Detected change in 'C:\\Users\\Felipe Pinto\\OneDrive\\Documentos\\Faculdade\\PUC-RIO\\pos-fullstack-basico\\controller.py', reloading - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:46:22,462] INFO _log() L96    * Restarting with stat - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:46:24,412] WARNING _log() L96    * Debugger is active! - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:46:24,418] INFO _log() L96    * Debugger PIN: 730-829-146 - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:46:35,887] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:46:35] "[33mDELETE /1 HTTP/1.1[0m" 404 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:46:42,232] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:46:42] "DELETE /obs/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:46:54,422] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:46:54] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:47:16,391] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:47:16] "[31m[1mPOST /obs/1 HTTP/1.1[0m" 405 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:47:22,155] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:47:22] "POST /obs HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:47:29,912] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:47:29] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:47:34,940] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:47:34] "POST /obs HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:47:43,456] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:47:43] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:47:49,701] WARNING del_observacao() L181  Erro ao consultar a observação ("'NoneType' object has no attribute 'deleted'",) - call_trace=C:\Users\Felipe Pinto\OneDrive\Documentos\Faculdade\PUC-RIO\pos-fullstack-basico\controller.py L181 
[2024-04-14 18:47:49,702] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:47:49] "[31m[1mDELETE /obs/3 HTTP/1.1[0m" 400 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:47:59,462] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:47:59] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:48:13,222] WARNING del_observacao() L181  Erro ao consultar a observação ("'NoneType' object has no attribute 'deleted'",) - call_trace=C:\Users\Felipe Pinto\OneDrive\Documentos\Faculdade\PUC-RIO\pos-fullstack-basico\controller.py L181 
[2024-04-14 18:48:13,223] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:48:13] "[31m[1mDELETE /obs/3 HTTP/1.1[0m" 400 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:48:57,609] WARNING del_observacao() L181  Erro ao consultar a observação ("'NoneType' object has no attribute 'deleted'",) - call_trace=C:\Users\Felipe Pinto\OneDrive\Documentos\Faculdade\PUC-RIO\pos-fullstack-basico\controller.py L181 
[2024-04-14 18:48:57,612] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:48:57] "[31m[1mDELETE /obs/3 HTTP/1.1[0m" 400 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:48:58,324] INFO _log() L96    * Detected change in 'C:\\Users\\Felipe Pinto\\OneDrive\\Documentos\\Faculdade\\PUC-RIO\\pos-fullstack-basico\\controller.py', reloading - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:49:00,351] INFO _log() L96    * Restarting with stat - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:49:02,354] WARNING _log() L96    * Debugger is active! - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:49:02,360] INFO _log() L96    * Debugger PIN: 730-829-146 - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:49:15,311] WARNING del_observacao() L181  Erro ao consultar a observação ("'NoneType' object has no attribute 'deleted'",) - call_trace=C:\Users\Felipe Pinto\OneDrive\Documentos\Faculdade\PUC-RIO\pos-fullstack-basico\controller.py L181 
[2024-04-14 18:49:15,315] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:49:15] "[31m[1mDELETE /obs/3 HTTP/1.1[0m" 400 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:49:27,201] INFO _log() L96    * Detected change in 'C:\\Users\\Felipe Pinto\\OneDrive\\Documentos\\Faculdade\\PUC-RIO\\pos-fullstack-basico\\controller.py', reloading - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:49:28,866] INFO _log() L96    * Restarting with stat - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:49:30,840] WARNING _log() L96    * Debugger is active! - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:49:30,846] INFO _log() L96    * Debugger PIN: 730-829-146 - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:49:44,054] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:49:44] "DELETE /obs/3 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:49:48,305] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:49:48] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:55:59,466] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:55:59,467] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:55:59,469] INFO _log() L96    * Restarting with stat - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:56:01,571] WARNING _log() L96    * Debugger is active! - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:56:01,578] INFO _log() L96    * Debugger PIN: 730-829-146 - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:56:02,835] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:56:02] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:56:39,036] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:56:39] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:57:38,514] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:57:38] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:58:04,391] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:58:04] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 18:58:21,949] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 18:58:21] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 19:00:00,290] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 19:00:00] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 19:06:06,548] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 19:06:06] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 19:06:49,505] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 19:06:49] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 19:07:58,794] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 19:07:58] "OPTIONS /obs HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 19:07:59,085] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 19:07:59] "POST /obs HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 19:08:00,393] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 19:08:00] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 19:08:06,516] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 19:08:06] "OPTIONS /obs HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 19:08:06,788] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 19:08:06] "POST /obs HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
[2024-04-14 19:08:07,535] INFO _log() L96   127.0.0.1 - - [14/Apr/2024 19:08:07] "GET /ordem/1 HTTP/1.1" 200 - - call_trace=c:\Python39\lib\site-packages\werkzeug\_internal.py L96  
